src/index.ts

```diff
@@ -246,6 +260,8 @@ export class UltravoxSession {
           this.state.addOrUpdateTranscript(newTranscript);
         }
       }
+    } else if (this.experimental) {
```

Comment:
what do experimental messages look like? what is in the payload? what are they used for?

Reply:
Happy to chat about any you need in specific, but these are deliberately unstructured in general.